lib/presentation/mixins/pill_cache_mixin.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:matrix/matrix.dart';
+
+mixin PillCacheMixin {
+  final Map<String, dynamic> mentionCache = {};
```

Comment:
a lot of place we cache for profile in memory. Not a safe way. please consistent it.